framework/src/modules/interoperability/base_cross_chain_update_command.ts

```diff
@@ -295,9 +328,8 @@ export abstract class BaseCrossChainUpdateCommand<
 			});
 		}
 	}
-	/**
-	 * @see https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md#bounce
-	 */
+
+	// https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md#bounce
```

Comment:
why we changed this to single comment? I thought we are following TypeDoc way to mention LIP references?